packages/grpc-js/test/test-call-credentials.ts

```diff
@@ -86,21 +86,16 @@ describe('CallCredentials', () => {
       const callCredentials = CallCredentials.createFromMetadataGenerator(
         generateFromServiceURL
       );
-      let metadata: Metadata;
-      try {
-        metadata = await callCredentials.generateMetadata({
-          service_url: 'foo',
-        });
-      } catch (err) {
-        throw err;
-      }
+      const metadata: Metadata = await callCredentials.generateMetadata({
```

Comment:
@murgatroid99  this is another material change; I removed the redundant try/catch (since the catch block just rethrows the caught error).

This was causing a lint error and seems redundant anyway.